src/as-content-rating.c

```diff
@@ -1203,39 +1195,50 @@ static const struct {
-	{ "sex-themes",		OARS_1_0, 0, 13, 14, 15 },
-	{ "language-profanity",	OARS_1_0, 0, 8, 11, 14 },
-	{ "language-humor",	OARS_1_0, 0, 3, 8, 14 },
-	{ "language-discrimination", OARS_1_0, 0, 9, 10, 11 },
-	{ "money-advertising",	OARS_1_0, 0, 7, 8, 10 },
-	{ "money-gambling",	OARS_1_0, 0, 7, 10, 18 },
-	{ "money-purchasing",	OARS_1_0, 0, 12, 14, 15 },
-	{ "social-chat",	OARS_1_0, 0, 4, 10, 13 },
-	{ "social-audio",	OARS_1_0, 0, 15, 15, 15 },
-	{ "social-contacts",	OARS_1_0, 0, 12, 12, 12 },
-	{ "social-info",	OARS_1_0, 0, 0, 13, 13 },
-	{ "social-location",	OARS_1_0, 0, 13, 13, 13 },
+	{ "violence-cartoon",	AS_OARS_VERSION_1_0, 0, 3, 4, 6 },
+	{ "violence-fantasy",	AS_OARS_VERSION_1_0, 0, 3, 7, 8 },
+	{ "violence-realistic",	AS_OARS_VERSION_1_0, 0, 4, 9, 14 },
+	{ "violence-bloodshed",	AS_OARS_VERSION_1_0, 0, 9, 11, 18 },
+	{ "violence-sexual",	AS_OARS_VERSION_1_0, 0, 18, 18, 18 },
+	{ "drugs-alcohol",	AS_OARS_VERSION_1_0, 0, 11, 13, 16 },
+	{ "drugs-narcotics",	AS_OARS_VERSION_1_0, 0, 12, 14, 17 },
+	{ "drugs-tobacco",	AS_OARS_VERSION_1_0, 0, 10, 13, 13 },
+	{ "sex-nudity",		AS_OARS_VERSION_1_0, 0, 12, 14, 14 },
+	{ "sex-themes",		AS_OARS_VERSION_1_0, 0, 13, 14, 15 },
+	{ "language-profanity",	AS_OARS_VERSION_1_0, 0, 8, 11, 14 },
+	{ "language-humor",	AS_OARS_VERSION_1_0, 0, 3, 8, 14 },
+	{ "language-discrimination", AS_OARS_VERSION_1_0, 0, 9, 10, 11 },
+	{ "money-advertising",	AS_OARS_VERSION_1_0, 0, 7, 8, 10 },
+	{ "money-gambling",	AS_OARS_VERSION_1_0, 0, 7, 10, 18 },
+	{ "money-purchasing",	AS_OARS_VERSION_1_0, 0, 12, 14, 15 },
+	{ "social-chat",	AS_OARS_VERSION_1_0, 0, 4, 10, 13 },
+	{ "social-audio",	AS_OARS_VERSION_1_0, 0, 15, 15, 15 },
+	{ "social-contacts",	AS_OARS_VERSION_1_0, 0, 12, 12, 12 },
+	{ "social-info",	AS_OARS_VERSION_1_0, 0, 0, 13, 13 },
+	{ "social-location",	AS_OARS_VERSION_1_0, 0, 13, 13, 13 },
 	/* v1.1 additions */
-	{ "sex-homosexuality",	OARS_1_1, 0, 13, 14, 15 },
-	{ "sex-prostitution",	OARS_1_1, 0, 12, 14, 18 },
-	{ "sex-adultery",	OARS_1_1, 0, 8, 10, 18 },
-	{ "sex-appearance",	OARS_1_1, 0, 10, 10, 15 },
-	{ "violence-worship",	OARS_1_1, 0, 13, 15, 18 },
-	{ "violence-desecration", OARS_1_1, 0, 13, 15, 18 },
-	{ "violence-slavery",	OARS_1_1, 0, 13, 15, 18 },
+	{ "sex-homosexuality",	AS_OARS_VERSION_1_1, 0, 13, 14, 15 },
+	{ "sex-prostitution",	AS_OARS_VERSION_1_1, 0, 12, 14, 18 },
+	{ "sex-adultery",	AS_OARS_VERSION_1_1, 0, 8, 10, 18 },
+	{ "sex-appearance",	AS_OARS_VERSION_1_1, 0, 10, 10, 15 },
+	{ "violence-worship",	AS_OARS_VERSION_1_1, 0, 13, 15, 18 },
+	{ "violence-desecration", AS_OARS_VERSION_1_1, 0, 13, 15, 18 },
+	{ "violence-slavery",	AS_OARS_VERSION_1_1, 0, 13, 15, 18 },
 };
 
-static gboolean
-is_oars_key (const gchar *id, OarsVersion version)
+/**
+ * as_is_oars_key:
+ * @id: the subsection ID e.g. `violence-cartoon`
+ * @version: the #AsOarsVersion, e.g. %AS_OARS_VERSION_1_1
+ *
+ * Checks if a OARS type exists in the given version.
+ *
+ * Returns: If the type exists.
+ *
+ * Since: 1.0
```

Comment:
Please drop the "Since" line, since this is internal API now (and it will make backporting the patch easier).